community/modules/compute/schedmd-slurm-gcp-v6-nodeset/variables.tf

```diff
@@ -29,6 +29,14 @@ variable "node_conf" {
   description = "Map of Slurm node line configuration."
   type        = map(any)
   default     = {}
+  validation {
+    condition     = lookup(var.node_conf, "Sockets", null) == null
+    error_message = <<-EOD
+    `Sockets` settings conflicts with `SocketsPerBoard` that is automatically computed by SlurmGCP.
```

Comment:
`s/settings/setting/`

Reply:
I believe this is addressed in the updated version but if not feel free to open this comment again